quest/__init__.py

```diff
@@ -6,10 +6,15 @@
 """
 from __future__ import (absolute_import, division,
                         print_function, unicode_literals)
-from builtins import *
 import pbr.version
-import os
+import warnings
 import logging
+import urllib3
+import os
+
+warnings.filterwarnings("ignore", message="numpy.dtype size changed")
```

Comment:
I don't get these warnings in my environments. I think we need to see how reproducible they are before officially ignoring these warnings here.

Reply:
I think pyviz is using an older version, sincee most of their stuff is a pinned version. Just a hunch.

Reply:
In that case I wouldn't ignore the warnings

Reply:
I just created a new environment and now I'm getting those numpy warnings again: 

```
/Users/rditlsc9/miniconda/envs/quest-py3/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
```
According to this issue: https://github.com/ContinuumIO/anaconda-issues/issues/6678 this is a Cython warning resulting as a result of some libraries being compiled against a different version of numpy than is currently installed, but they say that it can safely be ignored. I think we **should** filter this warning.